packages/backend/src/core/SearchService.ts

```diff
@@ -65,14 +65,17 @@ function compileQuery(q: Q): string {
 export class SearchService {
 	private readonly meilisearchIndexScope: 'local' | 'global' | string[] = 'local';
 	private meilisearchNoteIndex: Index | null = null;
-
+	private elasticsearchNoteIndex: string | null = null;
```

Comment:
```suggestion
	private elasticsearchNoteIndex: string | null = null;

```
メンバー変数の定義の下の改行は残してほしい